src/webui/btjson.cpp

```diff
@@ -800,6 +801,7 @@ QVariantMap toMap(BitTorrent::TorrentHandle *const torrent)
     ret[KEY_TORRENT_NUM_INCOMPLETE] = torrent->totalLeechersCount();
     const qreal ratio = torrent->realRatio();
     ret[KEY_TORRENT_RATIO] = (ratio > BitTorrent::TorrentHandle::MAX_RATIO) ? -1 : ratio;
+    ret[KEY_TORRENT_STATUS] = torrent->state().toString();
```

Comment:
I incline not sending the value twice, there should be a way to have `var status` point to the same value as `var state`.